Rust/Rust.sublime-syntax

```diff
@@ -238,9 +238,10 @@ contexts:
         1: punctuation.definition.annotation.rust
         2: punctuation.section.group.begin.rust
       push:
+        - meta_scope: meta.annotation.rust
```

Comment:
Yes, meta_scope gets applied to all of the matched text that causes the push, and it also gets applied to all of the matched text that causes the pop. In contrast, meta_content_scope doesn't have this property.